evaluate.py

```diff
@@ -67,19 +66,16 @@ def calculate_report(labels, decodings, distances, losses):
     '''
     samples = pmap(process_decode_result, zip(labels, decodings, distances, losses))
 
-    total_levenshtein = sum(s.levenshtein for s in samples)
-    total_label_length = sum(s.label_length for s in samples)
-
-    # Getting the WER from the accumulated levenshteins and lengths
-    samples_wer = total_levenshtein / total_label_length
+    # Getting the WER and CER from the accumulated edit distances and lengths
+    samples_wer, samples_cer = wer_cer_batch(labels, decodings)
```

Comment:
Is ``samples_cer`` really the CER of the samples?

Seems like it's more the average CER per word for the samples, which is slightly different from the CER for the samples, see my comments in ```wer_cer_batch()```

Maybe a renaming of ```wer_cer_batch()``` and ```samples_cer``` are in order?